spread.yaml

```diff
@@ -1480,4 +1480,47 @@ suites:
                 systemctl status snapd.socket || true
             fi
 
+    # a subset of the main test suite but for cross distro and with reexec enabled
+    tests/cross-distro-reexec/:
+        summary: Cross-distro reeexec
+        systems:
+          - arch-linux-*
+          # TODO: enable the following:
+          # - opensuse-*
+          # - fedora-*
+          # - amazon-linux-*
+        environment:
+          # enable snap rexec
+          SNAP_REEXEC: "1"
```

Comment:
Should we expand the language a little to say:

unset - use defaults different per distro
0 - never
1 - force if possible
2 - force even if broken

Shall we also add string values on top of the integers?

Reply:
at this point explicit SNAP_REEXEC=1 is the same as 'force' IIRC.